python/TestHarness/testers/CSVDiff.py

```diff
@@ -24,6 +27,13 @@ def __init__(self, name, params):
     def getOutputFiles(self):
         return self.specs['csvdiff']
 
+    # Check that override parameter lists are the same length
+#    def checkRunnable(self, options):
+#        if (len(self.specs['override_columns']) != len(self.specs['override_rel_err'])) or (len(self.specs['override_columns']) != len(self.specs['override_abs_zero'])) or (len(self.specs['override_rel_err']) != len(self.specs['override_abs_zero'])):
+#          self.addCaveats('Override inputs not the same length')
+#          self.setStatus(self.bucket_skip.status, self.bucket_fail)
+#          return False
```

Comment:
If you add a `return FileTester.checkRunnable(self, options)` at the end (at the same level as the if) it seems to work for me.
If you uncomment this block, be sure to fix the indentation as well.